Binary files boinc-client_release-7.4-7.4.42.orig/sched/cgi and boinc-client_release-7.4-7.4.42/sched/cgi differ
diff -u -r boinc-client_release-7.4-7.4.42.orig/sched/sched_send.cpp boinc-client_release-7.4-7.4.42/sched/sched_send.cpp
--- boinc-client_release-7.4-7.4.42.orig/sched/sched_send.cpp	2015-12-28 23:13:50.261961644 -0800
+++ boinc-client_release-7.4-7.4.42/sched/sched_send.cpp	2015-12-28 23:20:16.940919930 -0800
@@ -1424,7 +1424,7 @@
     //
     for (i=1; i<NPROC_TYPES; i++) {
         COPROC* cp = g_request->coprocs.proc_type_to_coproc(i);
-        if (cp->count) {
+        if (cp && cp->count) {
             g_wreq->req_secs[i] = clamp_req_sec(cp->req_secs);
             g_wreq->req_instances[i] = cp->req_instances;
             if (cp->estimated_delay < 0) {
Binary files boinc-client_release-7.4-7.4.42.orig/sched/sched_send.o and boinc-client_release-7.4-7.4.42/sched/sched_send.o differ
